doc/en/components/customize-marker-display-in-blazor-charts.md

```diff
@@ -0,0 +1,283 @@
+
+In other words, the value of each property of the SampleDataType record type can be referenced from the argument when calling the measure method. This way, the size of the marker is calculated based on the specific value and the call from the Ignite UI is answered.
+
+As a result, when calling this method the width and height of the marker (both in px) are set in the width and height fields of the argument.
+
+```razor
+// wwwroot / customMarkerTemplateFunc.js
+
+function customMarkerTemplateFunc ( ) {
+
+return {
+    measure: function ( measureInfo ) {
+
+    // In this example, based on the Volume property value of the data to draw
+
+    // 3 times that radius (so the diameter is 2 times that) Circle as a marker
+
+    // Width and height are calculated and set for drawing.
+
+    const item = measureInfo. data . item ;
+
+    const size = item. Volume * 3 * 2 ;
+
+    measureInfo. width = size;
+
+    easureInfo. height = size;
+}, 
+...
+```
+
+## Implementing a render Method for the Marker's Custom Rendering Object
+
+Next, we continue by implementing the render method. This method is called from the Ignite UI side every time a marker needs to be rendered. 
+During that time, the JavaScript-side representation of the data to be rendered will be included in the .data.item field of the argument object when this render method is called (similarly to the measure method). 
+
+Therefore, when calling the render method, the data to be drawn can be referenced as a marker drawing parameter via its JavaScript-side representation. 
+You will see that the method draws a marker on the 2D context object of the HTML Canvas element passed via that argument.
+
+```razor
+// wwwroot / customMarkerTemplateFunc.js
+
+function customMarkerTemplateFunc ( ) {
+
+return {
+
+    ... 
+
+    render: function ( renderInfo ) {
+
+    // Since the renderInfo passed as an argument is packed with coordinate-related information for drawing,
+
+    // Take this out
+
+    const cx = renderInfo.xPosition;
+
+    const cy = renderInfo.yPosition;
+
+    const halfWidth = renderInfo.availableWidth / 2.0;
+
+    const halfHeight = renderInfo.availableHeight / 2.0;
+
+    // For the marker fill color, use the Color property value of the data to be drawn.
+
+    // (By the way, the default marker fill color is
+
+    // stored in renderInfo.data.actualItemBrush.fill)
+
+    const color = renderInfo.data.item .Color; 
+
+    // Draw a marker against the 2D context of the HTML Canvas element
+
+    // (Draw a perfect circle marker with the size calculated by the measure method)
+
+    const ctx = renderInfo.context ; 
+
+    ctx.beginPath(); 
+
+    ctx.fillStyle = `rgba (${color.R} , ${color.G} , ${color.B} , ${color.A})`; 
+
+    ctx.ellipse (cx, cy, halfWidth, halfHeight, 0 , 0 , 360 * Math.PI / 180); 
+
+    ctx.fill(); 
+}
```

Comment:
```suggestion
function customMarkerTemplateFunc () {
    return {
        ... 
        render: function ( renderInfo ) {
            // Since the renderInfo passed as an argument is packed with coordinate-related information for drawing,
            // Take this out
            const cx = renderInfo.xPosition;
            const cy = renderInfo.yPosition;
            const halfWidth = renderInfo.availableWidth / 2.0;
            const halfHeight = renderInfo.availableHeight / 2.0;
            // For the marker fill color, use the Color property value of the data to be drawn.
            // (By the way, the default marker fill color is
            // stored in renderInfo.data.actualItemBrush.fill)
            const color = renderInfo.data.item .Color; 
            // Draw a marker against the 2D context of the HTML Canvas element
            // (Draw a perfect circle marker with the size calculated by the measure method)
            const ctx = renderInfo.context ; 
            ctx.beginPath(); 
            ctx.fillStyle = `rgba (${color.R} , ${color.G} , ${color.B} , ${color.A})`; 
            ctx.ellipse (cx, cy, halfWidth, halfHeight, 0 , 0 , 360 * Math.PI / 180); 
            ctx.fill(); 
        }
    }
}
```